src/core/Akka/Event/Logging.cs

```diff
@@ -91,7 +91,8 @@ public static ILoggingAdapter GetLogger(this IActorContext context, ILogMessageF
         {
             // if we're runing Akka.Remote or Akka.Cluster, this will grab the default inbound listening address
             // otherwise, it'll default to just using the local absolute path
-            var addr = ((ExtendedActorSystem) context.System).Provider.DefaultAddress ?? Address.AllSystems;
+            var provider = ((ExtendedActorSystem) context.System).Provider;
```

Comment:
yeah this ain't gonna fix it, based on your earlier comments @zbynek001 .... need to come up with some way to allow us to capture the self - address value if the actor being started is launched before the `RemoteActorRefProvider` finishes initializing...